modules/tile-converter/src/i3s-converter/attribute-metadata-info.ts

```diff
@@ -0,0 +1,246 @@
+import type {
+  Attribute,
+  AttributeStorageInfo,
+  ESRIField,
+  Field,
+  FieldInfo,
+  PopupInfo
+} from '@loaders.gl/i3s';
+
+import {AttributeType} from '@loaders.gl/i3s';
+
+export default class AttributeMetadataInfo {
```

Comment:
Nit: Prefer that we avoid export `default`. Named exports allow the transpiler to keep track of names.

Reply:
Done